service/create/service.go

```diff
@@ -232,21 +232,36 @@ func (s *Service) Boot() {
 					}
 
 					// Run masters
-					if err := s.runMachines(runMachinesInput{
+					masterIDs, err := s.runMachines(runMachinesInput{
 						clients:             clients,
 						spec:                cluster.Spec,
 						tlsAssets:           tlsAssets,
 						clusterName:         cluster.Name,
 						keyPairName:         keyPairName,
 						instanceProfileName: instanceProfileName,
 						prefix:              prefixMaster,
-					}); err != nil {
+					})
+
+					if err != nil {
+						s.logger.Log("error", microerror.MaskAny(err))
+						return
+					}
+
+					masterIP := masterIDs[0]
```

Comment:
I thought that the above error checking would be enough: if there's no master, there should be an error.

But I'll be defensive and an an explicit check for this.